### ISSUE TITLE
No installation guide.

### ISSUE THREAD

**Issue by belldandu** [ghyster/dns-angular#1]
Installing this is very tedious atm especially for people that have never actually used a php framework before. 
I suggest actually putting how to use this and how to install it in your readme instead of having nothing.

**Comment by ghyster** [comment]
you're right.I don't have time to work on this project now but in the next weeks this will be my first task, I'll write a proper readme with installation instructions

**Comment by belldandu** [comment]
Thank you.
